testbench/rest_server.py

```diff
@@ -979,6 +979,8 @@ def object_insert(bucket_name):
         blob, projection = gcs_type.object.Object.init_media(flask.request, bucket)
     elif upload_type == "multipart":
         blob, projection = gcs_type.object.Object.init_multipart(flask.request, bucket)
+
+    if upload_type == "media" or upload_type == "multipart":
```

Comment:
I think you can remove this if -- the `if resumable` block above has a return so all remaining uploads are either media or multipart.

Reply:
Done.